src/pages/biggive.html

```diff
@@ -483,10 +483,10 @@ <h2>This is the text which I want to add</h2>
 
         <biggive-heading  text="This is my title" size="4"></biggive-heading>
 
-        <biggive-social-icon-group>
```

Comment:
It looks to me like `biggive-icon-group` is not used anywhere at all, hence its deletion. The wrong selector was used here and Donate & WP seem to make no attempt to use it.